packages/ui/src/QueryProvider/index.tsx

```diff
@@ -9,12 +10,13 @@ type QueryProviderWrapperProps = {
 }
 
 export function QueryProvider({ children, persister, queryClient }: QueryProviderWrapperProps) {
+  const content = (<>{children}<ReactQueryDevtools /></>)
```

Comment:
According to [the documentation](https://tanstack.com/query/latest/docs/framework/react/devtools#install-and-import-the-devtools):
> _By default, React Query Devtools are only included in bundles when process.env.NODE_ENV === 'development', so you don't need to worry about excluding them during a production build._